### PR TITLE
QualityControlViewPane unit tests

### PR DESCRIPTION
**Pull request by antares1470**
### Description of the Change

Added unit tests for `QualityControlViewPane`. Also did a little code tidyup + made some adjustments in order to allow testing to occur

### Why Should This Be In Core?

Unit testing a class in Core

### Benefits

Better chance of catching out errors in the code

### Possible Drawbacks

-

### Applicable Issues

#1023

### CODE REVIEW

**Comment by aldebaran30701** [comment]
There looks to have been an issue with the CI - running tests. Would you be able to look into if this is an issue with your changes specifically?

File: CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/QualityControlViewPane.java [hunk 1]
```diff
@@ -680,46 +677,34 @@ private static void readSerializedRulePriorities() {
...
         }
         return rulePriorities;
     }
 
-    /**
-     * Lazily instantiates the lookup object
-     *
-     * @return the cached lookup object
-     */
-    public static Lookup getLookup() {
```
Comment (aldebaran30701) [inline CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/QualityControlViewPane.java:695]: I believe @arcturus2 mentioned that in the past there has been slowness issues with always fetching the lookup. I think this was added as a way to store the lookup object - Keen to hear if @arcturus2 thoughts are still the same or if it's a non-issue

Comment (arcturus2) [reply CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/QualityControlViewPane.java:695]: You've got a good memory @aldebaran30701. So there is definitely a performance issue making `Lookup` calls like `Lookup.getDefault().lookup(Something.class)` but `Lookup.getDefault()` should be fine as its just asking for the singleton version of `Lookup` which is OK.

Just to double check I profiled the 2 methods where `Lookup` was being called in this branch and can confirm its fine.

image

Comment (aldebaran30701) [reply CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/QualityControlViewPane.java:695]: Thanks for the detailed response! Continue as you were then :D

**Comment by arcturus2** [comment]
Regarding the issue with the unit test not completing I think the issue might be related to the calls to `new JFXPanel()` or something else that requires graphics to be enabled. I am going to look at introducing Xvfb (https://en.wikipedia.org/wiki/Xvfb) into the build image to see if that will help resolve the issue.

**Review by aldebaran30701 — APPROVED** [review]

**Review by arcturus2 — APPROVED** [review]

**Pull request merged**